src/Meziantou.Framework.CodeOwners/CodeOwnersSection.cs

```diff
@@ -5,14 +5,21 @@ namespace Meziantou.Framework.CodeOwners;
 [StructLayout(LayoutKind.Auto)]
 public readonly struct CodeOwnersSection : IEquatable<CodeOwnersSection>
 {
-    public CodeOwnersSection(string name, bool isOptional)
+    public CodeOwnersSection(string name, int requiredReviewerCount = 1, IReadOnlyCollection<string> defaultOwners = null)
```

Comment:
Can you make it `internal`? I don't think it need to be public.

Reply:
Sure, addressed 👌